habitat_sim/utils/data/data_extractor.py

```diff
@@ -245,6 +249,18 @@ def _generate_label_map(self, scene, verbose=False):
 
         return instance_id_to_name
 
+    def _register_pose_extractor(self, method, args):
```

Comment:
We currently have a registry pattern implemented by `habitat_sim/registry.py`.  I suggest using that, that way users can easily ad their own pose extractor.

Reply:
Wasn't 100% sure how the registry system worked, but I did my best to figure it out. Let me know if there's anything that jumps out that needs changing.